app/views/branches/_validations.html.erb

```diff
@@ -0,0 +1,38 @@
+<% if @branch.errors.present? %>
```

Comment:
What happens if the user selects all properties from the branch but not from the conditionals? Like the default next is selected (from branch object) but not the conditional (conditional object)?